pkg/diagnostics/log/log.go

```diff
@@ -0,0 +1,197 @@
+package log
```

Comment:
Since you have settings, you should be passing through instances of Log on your options object.  In delegation cases, I'm going to want strong control over exactly what comes out.  This the second biggest failing of `glog`.